src/liveEditor/components/icons/fields/index.ts

```diff
@@ -1,17 +1,12 @@
+import { EditIcon } from "..";
 import { FieldDataType } from "../../../utils/types/index.types";
-import { UrlIcon } from "../url";
-import { CustomFieldIcon } from "./custom";
-import { HtmlRteIcon } from "./html-rte";
-import { ISODateIcon } from "./isodate";
-import { JsonRteIcon } from "./json-rte";
-import { MarkdownRteIcon } from "./markdown-rte";
 
 // SVG components for use with Preact components
 export const fieldIcons: Partial<Record<FieldDataType, React.FC>> = {
-    link: UrlIcon,
-    json_rte: JsonRteIcon,
-    html_rte: HtmlRteIcon,
-    markdown_rte: MarkdownRteIcon,
-    custom_field: CustomFieldIcon,
-    isodate: ISODateIcon,
+    link: EditIcon,
```

Comment:
This change is not required now, right? Since, we are using a different map now. We can even remove `fieldIcons` I guess and only keep the JSX icons, add remaining ones later on.

Reply:
We will take this up later, when we refactor icon related code.